### ISSUE TITLE
Clarify "writing a bit mask" for DMA:INTS0 and DMA:INTS1

### ISSUE THREAD

**Issue by dhalbert** [raspberrypi/pico-feedback#186]
The register doc tables for `DMA:INTS0` and `DMA:INTS1` say:

> Channel interrupts can be cleared by writing a bit mask here.

This is a wee bit vague, since it doesn't say whether you should write a 0 or a 1 to clear the interrupt. The SDK code shows you should write a 1.

**Comment by daveythacher** [comment]
> WC
This is a single bit that is typically set by a piece of hardware and then written to by the processor to clear the bit. The
bit is cleared by writing a 1, using either a normal write or the clear alias. See Section 2.1.2 for more information about
the clear alias.

**Comment by lurch** [comment]
Yes, see Appendix A of https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf
Closing this.
